src/components/teleporter.js

```diff
@@ -231,7 +198,9 @@ AFRAME.registerComponent("teleporter", {
         return;
       }
 
-      this.teleport();
+      this.characterController =
```

Comment:
Hmm, I would think this would be safe to resolve in init, at least the entity itself. And insofar as this remains a component that lives on an entity we maybe might as well use a schema property for this. Not particularly fond of it, but also don't love these self-caching queryselectors either so...